embark/uploader/management/commands/runapscheduler.py

```diff
@@ -0,0 +1,98 @@
+# runapscheduler.py
```

Comment:
maybe extend this module by realtime data using redis. (live graphs on dashboard)

Reply:
Yeah this could easily be done.